changelogs/fragments/azure_rm_appgateway-probe.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
do we actually need changelog fragment here?

Reply:
yes, actually seems like it was not enforced too much before, nor documented, but we should add doc fragments now

Reply:
Hi, just for info. We are discussing the cases when a changelog is and updating CI to error if it isn't present.
Generally speaking if you are updating something apart from a `./test/` or a typo in documentation a changelog fragment is useful, and makes it easier to backport fixes as changelogs are required there.